validate/validate_io.py

```diff
@@ -210,7 +210,7 @@ def create_checksum_file(path, hash_type):
         if hash_type == 'md5-gen':
             file.write(generate_md5(path))  
```

Comment:
Looks like `md5` generation has the same issue?

Reply:
I can fix that in this PR too